enterprise_subsidy/apps/api/schema.py

```diff
@@ -8,12 +8,61 @@
 from .v1.serializers import CanRedeemResponseSerializer, ExceptionSerializer
 
 
+class QueryParams:
+    """
+    A namespace class to hold constants describing the query parameter
+    strings allowed for a given view.
+    """
+    lms_user_id = 'lms_user_id'
+    content_key = 'content_key'
+    subsidy_access_policy_uuid = 'subsidy_access_policy_uuid'
+
+
+class LmsUserIdParameter(OpenApiParameter):
+    def __init__(self, **kwargs):
+        options = {
+            'type': OpenApiTypes.INT,
+            'location': OpenApiParameter.QUERY,
+            'required': True,
+            'allow_blank': False,
+            'description': "The user identifier to whom the query pertains.",
+        }
+        options.update(kwargs)
+        return super().__init__(QueryParams.lms_user_id, **options)
+
+
+class ContentKeyParameter(OpenApiParameter):
+    def __init__(self, **kwargs):
+        options = {
+            'type': OpenApiTypes.STR,
+            'location': OpenApiParameter.QUERY,
+            'required': True,
+            'allow_blank': False,
+            'description': "The content key/identifier to which the query pertains.",
+        }
+        options.update(kwargs)
+        return super().__init__(QueryParams.content_key, **options)
+
+
+class SubsidyAccessPolicyUuidParameter(OpenApiParameter):
+    def __init__(self, **kwargs):
+        options = {
+            'type': OpenApiTypes.STR,
+            'location': OpenApiParameter.QUERY,
+            'required': True,
+            'allow_blank': False,
+            'description': "The uuid of the subsidy access policy to which the query pertains.",
+        }
+        options.update(kwargs)
+        return super().__init__(QueryParams.subsidy_access_policy_uuid, **options)
+
+
 class Parameters:
```

Comment:
Parameters class is dead code now, right?

Reply:
`Parameters` still is used by the v1 `can_redeem` endpoint.  I removed the code above, though, once I found a way to utilize `FilterSet` in a way that drf-spec can usefully introspect.